packages/select/src/components/multi-select/multiSelect2.tsx

```diff
@@ -29,6 +29,7 @@ import {
     TagInputProps,
 } from "@blueprintjs/core";
 import { Popover2 } from "@blueprintjs/popover2";
+import { PopupKind } from "@blueprintjs/popover2/src/popupKind";
```

Comment:
is it not exported? sorry, that's a bug. feel free to add an import from `index.ts`